ui/app/scripts/shared/services/parser.js

```diff
@@ -901,7 +973,7 @@ define(function() {
 	        		}
 	        	}
         	}
-        	$log.info('JSParse: final lines: '+JSON.stringify(lines));
+        	// $log.info('JSParse: final lines: '+JSON.stringify(lines));
```

Comment:
Yep, the parser (and whole system in general really) is producing too much output so I am starting to reduce it.